docs/install/index.md

```diff
@@ -65,36 +65,37 @@ Indicate your preferred configuration. Then, follow the customized commands to i
   <div class="python">
     <div class="cpu">
 
-The following installation instructions have been tested on Ubuntu 14.04 and 16.04.
+The following installation instructions have been tested on Ubuntu 14.04, 16.04 and 17.04
 
 <div class="virtualenv">
 <br/>
 
-**Step 1**  Install virtualenv for Ubuntu.
+**Step 1**  Install virtualenv for Ubuntu and required runtime libraries.
 
 ```bash
 $ sudo apt-get update
-$ sudo apt-get install -y python-dev python-virtualenv
+$ sudo apt-get install -y python3 python3-pip python3-dev python-virtualenv libgfortran3
 ```
 
 **Step 2**  Create and activate virtualenv environment for MXNet.
 
-Following command creates a virtualenv environment at `~/mxnet` directory. However, you can choose any directory by replacing `~/mxnet` with a directory of your choice.
+Following command creates a python3 virtualenv environment at `~/py3` directory. You can choose a different path for that.
 
 ```bash
-$ virtualenv --system-site-packages ~/mxnet
+$ virtualenv --system-site-packages -p /usr/bin/python3 ~/py3
```

Comment:
Can you update the env name to `mxnet_p36`? It would help keep some new docs in alignment so we can use the same examples. Also if you end up mentioning how to create a 2.7 env, then use `mxnet_p27`.